watertap/core/util/initialization.py

```diff
@@ -87,6 +87,8 @@ def check_dof(blk, fail_flag=False, logger=_log, expected_dof=0):
 
     """
     if degrees_of_freedom(blk) != expected_dof:
+        # can fix by initizalizing msg or can disable message
+        msg = "Message failed to initialize."
```

Comment:
I would probably just replace the else clause with a generic WaterTAPDeveloperError because the issue is that the else clause is calling a message that hasn't necessarily been initialized yet, right?

Reply:
I remember there was a reason we did a msg here and not an error or log ... I think